nxc/protocols/smb.py

```diff
@@ -309,10 +309,9 @@ def print_host_info(self):
         return True
 
     def kerberos_login(self, domain, username, password="", ntlm_hash="", aesKey="", kdcHost="", useCache=False):
-        logging.getLogger("impacket").disabled = True
```

Comment:
any raison ?

Reply:
At some point it got added to the logger.py file, so it's unnecessary here:
https://github.com/Pennyw0rth/NetExec/blob/45a9121bb12c705c75c0b351e5f93ee56b985c5a/nxc/logger.py#L97